hcl/fmt.go

```diff
@@ -442,9 +562,10 @@ func trimNewlines(tokens hclwrite.Tokens) hclwrite.Tokens {
 		}
 	}
 
-	if end < start {
-		return nil
-	}
+	// TODO(katcipis): assess if we really need this check
```

Comment:
if the fuzzer didnt catch anything here, maybe it's safe to remove this, right?

Reply:
Ah yeah, good catch, will remove.